var/spack/repos/builtin/packages/ecmwf-atlas/package.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+import os
+
+from spack.package import *
+
+
+class EcmwfAtlas(CMakePackage):
+    """A library for numerical weather prediction and climate modelling."""
+
+    homepage = "https://software.ecmwf.int/wiki/display/atlas"
+    git = "https://github.com/ecmwf/atlas.git"
+    url = "https://github.com/ecmwf/atlas/archive/0.22.1.tar.gz"
+
+    maintainers("climbfuji", "srherbener")
+
+    version("master", branch="master")
+    version("develop", branch="develop")
+    version("0.34.0", sha256="48536742cec0bc268695240843ac0e232e2b5142d06b19365688d9ea44dbd9ba")
+    version("0.33.0", sha256="a91fffe9cecb51c6ee8549cbc20f8279e7b1f67dd90448e6c04c1889281b0600")
+    version("0.32.1", sha256="3d1a46cb7f50e1a6ae9e7627c158760e132cc9f568152358e5f78460f1aaf01b")
+    version("0.31.1", sha256="fa9274c74c40c2115b9c6120a7040e357b0c7f37b20b601b684d2a83a479cdfb")
+    version("0.31.0", sha256="fa4ff8665544b8e19f79d171c540a9ca8bfc4127f52a3c4d4d618a2fe23354d7")
+
+    depends_on("ecbuild", type=("build"))
+    depends_on("eckit@:1.23", when="@:0.33")
+    depends_on("eckit@1.24:", when="@0.34:")
+    depends_on("boost cxxstd=14 visibility=hidden", when="@0.26.0:0.33.99", type=("build", "run"))
+    depends_on("boost cxxstd=17 visibility=hidden", when="@0.34.0:", type=("build", "run"))
+    variant("fckit", default=True, description="Enable fckit")
+    depends_on("fckit@:0.10", when="@:0.33 +fckit")
+    depends_on("fckit@0.11:", when="@0.34: +fckit")
+    depends_on("python")
+
+    patch("clang_include_array.patch", when="%apple-clang")
+    patch("clang_include_array.patch", when="%clang")
+    patch("intel_vectorization_v0p33.patch", when="@:0.33 %intel")
+    patch("intel_vectorization_v0p34.patch", when="@0.34: %intel")
+
+    variant(
+        "build_type",
+        default="RelWithDebInfo",
+        description="CMake build type",
+        values=("Debug", "Release", "RelWithDebInfo"),
+    )
+
+    variant("openmp", default=True, description="Use OpenMP?")
+    depends_on("llvm-openmp", when="+openmp %apple-clang", type=("build", "run"))
+    variant("shared", default=True, description="Build shared libraries")
+
+    variant("trans", default=False, description="Enable trans")
+    depends_on("ectrans@1.1.0:", when="@0.31.0: +trans")
+    # variant('cgal', default=False)
+    # depends_on('cgal', when='+cgal')
```

Comment:
```suggestion
```

Reply:
I am against removing this, because it takes time to go through CMakeLists.txt and check with options are available in principle. I don't want to do this again, therefore the reminder that cgal is an option that we simply haven't implemented yet.